SU2_PY/discrete_adjoint.py

```diff
@@ -42,27 +42,30 @@ def main():
                       help="read config from FILE", metavar="FILE")
     parser.add_option("-n", "--partitions", dest="partitions", default=1,
                       help="number of PARTITIONS", metavar="PARTITIONS")
-    parser.add_option("-c", "--compute",    dest="compute",    default="True",
-                      help="COMPUTE direct and adjoint problem", metavar="COMPUTE")
     parser.add_option("-s", "--step",       dest="step",       default=1E-4,
                       help="DOT finite difference STEP", metavar="STEP")
     parser.add_option("-v", "--validate", dest="validate", default="False",
                       help="Validate the gradient using direct diff. mode", metavar="VALIDATION")
     parser.add_option("-z", "--zones", dest="nzones", default="1",
                       help="Number of Zones", metavar="ZONES")
+    parser.add_option("-m", "--mode", dest="mode", default="0",
```

Comment:
I would suggest using a string name instead of an int for the mode options, just to be more readable

Reply:
Yep, that would be also my only remark! Otherwise looks good.

Reply:
Okay, I've updated it.